recipes/openconnect-sso/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "openconnect-sso" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vlaci/openconnect-sso/releases/download/v{{ version }}/openconnect-sso-{{ version }}.tar.gz
+  sha256: a7bde0a8dd35d7bab3c66f28b420c343c270a7ee45e98e04bfa6ecc6958f5c5b
+
+build:
+  skip: True  # [not unix]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  build:
+    - poetry >=0.12
+  host:
+    - python
+    - pip
+    - poetry >=0.12
+  run:
+    - python
+    - attrs >=18.2
+    - {{ pin_compatible("colorama", lower_bound="0.4") }}
```

Comment:
I don't really understand this syntax. why not just set a standard `>=` constraint?

Reply:
This is to mirror their [`pyproject.yaml`](https://github.com/vlaci/openconnect-sso/blob/master/pyproject.toml) which says
```
colorama = "^0.4"
```
In the poetry syntax this translates to `colorama >=0.4,<1.0`. The `pin_compatible` syntax IIUC is doing the same job. I don't have strong preference and will do whatever requested by reviewers if I can get this merged sooner 🙂

Reply:
Typically pin compatible is used if there is a pin in the host section. Please use the other syntax

Reply:
Done in https://github.com/conda-forge/staged-recipes/commit/4751cd40b085f9a5ade1a2b5ae7d215d58da5836.